### PR TITLE
esa-snap: add German page

### PR DESCRIPTION
**Pull request by sebfoe**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution,

@sebfoe Can you add an English page for this command along with the German page? Also, We already have `snap` for Linux (not this command but a Universal package manager for Linux) if possible can you rename this page to something like "esa-snap"? I will ask other's opinions on this matter. Also, can you link to a more specific documentation page?

File: pages.de/linux/snap.md
Comment (marchersimon) [inline pages.de/linux/snap.md:4]: `Weitere Informationen` is just a standard we have.
And the new link is a bit more specific, even if it's a download link.
```suggestion
> Weitere Informationen: <http://step.esa.int/main/download/snap-download/>.
```

File: pages.de/linux/snap.md
Comment (marchersimon) [inline pages.de/linux/snap.md:6]: We write descriptions in imperative mood. Also for the `--help flag`, we usually use something like this:
```suggestion
- Zeige Hilfe an:
```

File: pages.de/linux/snap.md
Comment (marchersimon) [inline pages.de/linux/snap.md:10]:
```suggestion
- Zeige alle Updates an:
```

**Review by marchersimon — COMMENTED** [review]
As @kbdharun already said, we should always start with the English page. This case is a bit tricky, because if we rename this page to `esa-snap`, nobody would find it. We could add a line in `snap.md` saying `for the ESA snap run tldr esa-snap`, but then again we would "pollute" the page for a very popular command with a command very few people use.

Any better ideas?

**Comment by sebfoe** [comment]
> As @kbdharun already said, we should always start with the English page. This case is a bit tricky, because if we rename this page to `esa-snap`, nobody would find it. We could add a line in `snap.md` saying `for the ESA snap run tldr esa-snap`, but then again we would "pollute" the page for a very popular command with a command very few people use.
> 
> Any better ideas?

@kbdharun, thank you very much for your support! I will add/create the English version.
@marchersimon thank you very much for adding a more specific link!
@kbdharun, @marchersimon, the snap package manager.
Atm, 'esa-snap' would be my prefered suggestion, the package manager 'snap' is a much more important tool.
Of course not in the opinion of a remote sensing user, but its about quantity here, I would say.
Any better ideas are highly welcome, thank you!

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — CHANGES_REQUESTED** [review]
Blocking this PR from being merged.

Note to maintainers: Wait for #9334 to be merged and then review this PR.

File: pages.de/linux/esa-snap.md
Comment (kbdharun) [inline pages.de/linux/esa-snap.md:6]: Move this command to last and this PR is GTG.

**Review by kbdharun — APPROVED** [review]

File: pages.de/linux/esa-snap.md
Comment (kbdharun) [inline pages.de/linux/esa-snap.md:12]:
```suggestion
# snap

> Sentinel Application Platform(SNAP) für die Prozessierung von Satellitendaten der Europäischen Raumfahrtagentur (ESA).
> Weitere Informationen: <http://step.esa.int/main/download/snap-download/>.

- Zeige alle Updates an:

`snap --nosplash --nogui --modules --list --refresh`

- Zeige Hilfe an:

`snap --help`
```

**Pull request merged**
